### PR TITLE
Add results directory arg to dotnet test

### PR DESCRIPTION
**Pull request by smadala**
- Add results directory parameter as first class parameter
-  Related Issue: Microsoft/vstest#299

### CODE REVIEW

**Comment by piotrpMSFT** [comment]
@dotnet-bot test RHEL7.2 x64 Release Build please.

File: src/dotnet/commands/dotnet-test/LocalizableStrings.cs [hunk 1]
```diff
@@ -55,5 +55,7 @@ internal class LocalizableStrings
 
         public const string RunSettingsArgsHelpText = @"Any extra commandline runsettings arguments that should be passed to vstest. See 'dotnet vstest --help' for available options.
                                         Example: -- RunConfiguration.ResultsDirectory=""C:\users\user\desktop\Results Directory"" MSTest.DeploymentEnabled=false";
+        public const string CmdResultsDirectoryDescription = @"Test results directory will be created in specified path if not exists.
+                                        Example: --results-directory <PATH_TO_RESULTS_DIRECTORY>";
```
Comment (jonsequitur) [inline src/dotnet/commands/dotnet-test/LocalizableStrings.cs]: I'd suggest a slight change to the message text: "The test results directory will be created in the specified path if it does not exist."

**Comment by piotrpMSFT** [comment]
@livarcocc merging this one next

**Comment by livarcocc** [comment]
@piotrpMSFT waiting for it to turn green.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OpenSUSE13.2 x64 Debug Build please.

**Pull request merged**